nexus/src/nexus.rs

```diff
@@ -1686,8 +1686,12 @@ impl Nexus {
             });
         }
 
-        let nics: Vec<external::NetworkInterface> = self
+        let nics = self
             .db_datastore
+            .derive_guest_network_interface_info(&opctx, &authz_instance)
+            .await?;
+
+        /*
```

Comment:
Should we remove this dead code?

Reply:
Yeah, caught this in a commit I haven't yet pushed, thanks!